tests/test_venue_request_v2.py

```diff
@@ -550,7 +550,19 @@ def test_venue_recruitment(self, client, test_client, selenium, request_page, ve
             readers=['{}/Program_Chairs'.format(venue['venue_id']), venue['support_group_id']],
             signatures=['~SomeFirstName_User1'],
             writers=[]
-        ))
+        )
+
+        with pytest.raises(openreview.OpenReviewException, match=r'Invalid token: .* does not exist.'):
```

Comment:
Why do you use a .* here instead of the actual string that makes the validation fail?

Reply:
This is because a variable is used in the error message to display what the invalid token is.

https://github.com/openreview/openreview-py/blob/bbb1b017ee936a7ce90c8d72c6629fb965882c29/openreview/venue_request/process/recruitment_pre_process.js#L16-L18

Reply:
Can you know the value of the token at test time?

Reply:
I see what you mean now, I made the change.

https://github.com/openreview/openreview-py/blob/45be3dc8de183e2807aadaad98fe08455a3bc286/tests/test_venue_request_v2.py#L555-L561